ansible-test/my-playbook.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: local-test
+  remote_user: root
+  tasks:
+    - name: add some config the .bashrc
+      become: yes
+      blockinfile:
+        path: /home/qujiabao/.bashrc
+        block: |
+          export DOCKER_HOST="0.0.0.0:2376"
```

Comment:
`0.0.0.0` means to bind this service to all local ipv4 addresses on this machine.

You probably ended up with this setting from some tutorial. But in this case we SHOULD be specific about which network interface to bind to.

I can explain in greater detail when we sync.